packages/botfuel-dialog/src/nlu-resolver.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2017 - present, Botfuel (https://www.botfuel.io).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const logger = require('logtown')('NluResolver');
+const Resolver = require('./resolver');
+
+/**
+ * The adapter resolver resolves the adapter at startup.
+ */
+class NluResolver extends Resolver {
+  /**
+   * @constructor
+   * @param {Object} bot - the bot
+   */
+  constructor(bot) {
+    super(bot.config, 'nlu');
+    this.bot = bot;
+  }
+
+  /** @inheritdoc */
+  getPaths(name) {
+    logger.debug('getPaths', name);
+    return [`${this.path}/${name}-${this.kind}.js`, `${__dirname}/${name}-${this.kind}.js`];
```

Comment:
is this correct?

Reply:
I think yes. `kindSdkPath` is `.../botfuel-dialog/src/nlus` which does not exist, that's why I use `__dirname` instead.

Reply:
replace `__dirname ` by `kindSdkPath`

Reply:
done